Integrations/IntSight/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+- Indicators issue fixed - were not extracted correctly in `!intsights-get-iocs`
```

Comment:
maybe:
```suggestion
- Fixed an issue where indicators were not extracted correctly in ***intsight-get-iocs*** command
```
@kirbles19 WDYT?

Reply:
Add a period to the end, and it's perfect.

Reply:
so close...